### PR TITLE
Use block characters to render

### PR DESCRIPTION
**Pull request by Lonami**
It's prettier.

### CODE REVIEW

**Comment by Lonami** [comment]
Note that this will fail to render an odd number of rows since `.zip()` takes the shortest sequence. What should we do about that?

File: src/lib.rs [hunk 1]
```diff
@@ -43,11 +43,14 @@ impl Grid {
     }
 
     /// Get an ASCII-art representation of the whole grid, with lines to indicate the edges.
+    ///
+    /// Note: the representation uses half the height since two pixels fit in per character.
     pub fn stringify(&self) -> String {
         let mut lines = Vec::with_capacity(self.len() + 2);
```
Comment (expectocode) [inline src/lib.rs]: pls to reduce allocation

File: src/main.rs [hunk 2]
```diff
@@ -4,7 +4,7 @@ use grid::Grid;
 use std::{thread, time};
 
 const WIDTH: usize = 80;
-const HEIGHT: usize = 20;
+const HEIGHT: usize = 40; // N pixels render in half character rows
```
Comment (expectocode) [inline src/main.rs]: don't worry about changing these, I'll make the edits about what the parameters mean

**Comment by Lonami** [comment]
I guess you were too impatient. Less work for me.

File: src/lib.rs [hunk 3]
```diff
@@ -66,13 +70,16 @@ impl Grid {
 }
 
 /// Get an ASCII-art representation of a single line, intended for use with `Grid.stringify()`.
```
Comment (expectocode) [inline src/lib.rs]: "ASCII-art"

... oops

Comment (Lonami) [reply src/lib.rs]: What about it?

Comment (expectocode) [reply src/lib.rs]: block drawing characters aren't in ascii

Comment (Lonami) [reply src/lib.rs]: Well, fixing the comment shouldn't be hard. Maybe add a test to ensure it prints non-ASCII from now!

Comment (expectocode) [reply src/lib.rs]: > Maybe add a test to ensure ...

haha good one

**Pull request merged**